src/Components/DonationFormModal.js

```diff
@@ -3,8 +3,8 @@ import '../Styles/DonationFormModal.scss';
 
 const DonationFormModal = () => {
   const [formState, setFormState] = useState({
-    firstName: '',
-    lastName: '',
+    ownerName: '',
```

Comment:
Nice catch seeing we needed a pivot to match the backend data

Reply:
Ditto. Recognized that first AND last name aren't what we'd decided as a team, so we fixed that here to simply ownerName rather than firstName and lastName